docs/quick-starts/framework/next/README.mdx

```diff
@@ -92,6 +92,48 @@ export const config = {
 Check the [next-sample](https://github.com/logto-io/js/tree/master/packages/next-sample) to see full example.
 :::
 
+## Use external session storage \{#use-external-session-storage}
```

Comment:
It seems these two parts are completely identical. So we'd better create a sharable mdx fragment and import it into these two guides.

Reply:
updated